network/wsNetwork.go

```diff
@@ -1830,6 +1830,11 @@ func (wn *WebsocketNetwork) sendPeerConnectionsTelemetryStatus() {
 			connDetail.Address = peer.OriginAddress()
 			connectionDetails.IncomingPeers = append(connectionDetails.IncomingPeers, connDetail)
 		}
+		rttInfo, err := util.GetConnRTT(peer.conn.UnderlyingConn())
+		if err == nil {
+			connDetail.RTT = int64(rttInfo.RTT)
+			connDetail.RTTVar = int64(rttInfo.RTTVar)
```

Comment:
Does something break if you use the same type (`uint32`) for these in `PeerConnectionDetails`?